tests/unit/behaviors/TranslatableModelTest.php

```diff
@@ -74,7 +74,7 @@ public function testGetTranslationValueNoFallback()
 
         $this->assertEquals('Australia', $obj->name);
 
-        $obj->noFallbackLocale()->translateContext('fr');
```

Comment:
Please, keep the tests for the deprecated method (until it actually gets removed).